.github/workflows/multi-platform-build-and-publish.yml

```
@@ -120,21 +110,20 @@ jobs:
-      keychain_name: signing.keychain-db # optional
+      metadata_path: './fastlane/metadata'
+      use_cocoapods: true
+      shared_module: ':cmp-shared'
       distribute_ios_firebase: ${{ inputs.distribute_ios_firebase }}
-      distribute_ios_testflight: ${{ inputs.distribute_ios_testflight }}
-      distribute_ios_appstore: ${{ inputs.distribute_ios_appstore }}
+      # When App Store is enabled, skip TestFlight — deliver() already uploads
+      # the IPA to App Store Connect which makes the build available in TestFlight.
+      # Running both in parallel causes version conflicts since the upstream
+      # reusable workflow has no job dependency between the two.
       distribute_macos_testflight: ${{ inputs.distribute_macos_testflight }}
```

actionlint (1.7.11)

[error] property "distribute_macos_testflight" is not defined in object type {distribute_ios_appstore: bool; distribute_ios_firebase: bool; distribute_ios_testflight: bool; release_type: string; target_branch: string}

(expression)